wenet/transformer/asr_model.py

```diff
@@ -129,13 +129,36 @@ def _calc_att_loss(
         )
         return loss_att, acc_att
 
-    @torch.jit.export
+    def _forward_encoder(
+        self,
+        speech: torch.Tensor,
+        speech_lengths: torch.Tensor,
+        decoding_chunk_size: int = -1,
+        num_decoding_left_chunks: int = -1,
+        simulate_streaming: bool = False,
+    ) -> Tuple[torch.Tensor, torch.Tensor]:
+        # Let's assume B = batch_size
+        # 1. Encoder
+        if simulate_streaming and decoding_chunk_size > 0:
+            encoder_out, encoder_mask = self.encoder.forward_chunk_by_chunk(
```

Comment:
Does num_decoding_left_chunks also work for forward_chunk_by_chunk？

Reply:
Yes, and it will be added in another PR to keep this PR simple